jobs/mongodb_migration/src/mongodb_migration/collector.py

```diff
@@ -330,4 +330,13 @@ def get_migrations(self) -> list[Migration]:
                 version="20240221160700",
                 description="merge 'split-first-rows-from-streaming' and 'split-first-rows-from-parquet' responses to 'split-first-rows'",
             ),
+            CacheDeletionMigration(
+                cache_kind="split-duckdb-index",
+                version="20240221134400",
```

Comment:
could you increase this value a bit, so that it's higher than the previous migration? 🙏  Our PRs were created concurrently

Reply:
Done